pkgs/by-name/bc/bc-decaf/package.nix

```diff
@@ -8,7 +8,7 @@
 
 stdenv.mkDerivation {
   pname = "bc-decaf";
-  version = "unstable-2022-07-20";
+  version = "linphone-5.2.6";
```

Comment:
Uhm so, that change was really familiar and kept nagging at me until I remembered [this comment](https://github.com/NixOS/nixpkgs/pull/220420#issuecomment-1463752372)

So.. hello again @felixsinger it seems we've had this conversation before, sorry I didn't remember sooner :sweat_smile: I would've realized you knew about the update procedure!

Anyway, back to that change, don't you want to use `unstable` naming instead?